wp-plugin-reviews.php

```diff
@@ -81,9 +81,11 @@ public function get_plugin_reviews($plugin, $count = 5) {
         }
 
         $key = self::CACHE_KEY_SLUG . $plugin;
+
         if (false === ( $output = get_transient( $key ) ) ) {
             require_once(ABSPATH . WPINC . '/feed.php');
-            $rss = fetch_feed(self::REVIEW_BASE_URL . $plugin);
+            $feed_url = 'https://wordpress.org/support/plugin/' . $plugin . '/reviews/feed/';

```

Comment:
Don't hardcode the url here. Change the `REVIEW_BASE_URL` constant.

Reply:
Fixed.